app/components/forms/user-payment-info-form.js

```diff
@@ -0,0 +1,111 @@
+import Component from '@ember/component';
+import FormMixin from 'open-event-frontend/mixins/form';
+import { validPhoneNumber } from 'open-event-frontend/utils/validators';
+import { pick, orderBy } from 'lodash-es';
+import { action, computed } from '@ember/object';
+import { countries } from 'open-event-frontend/utils/dictionary/demography';
+
+export default class extends Component.extend(FormMixin) {
+  didInsertElement() {
+    super.didInsertElement(...arguments);
+    this.set('userBillingInfo', pick(this.authManager.currentUser, ['billingContactName', 'billingCity', 'billingPhone', 'company', 'billingTaxInfo', 'billingCountry', 'billingState', 'billingAddress', 'billingZipCode', 'billingAdditionalInfo']));
+  }
+
+  getValidationRules() {
+    return {
+      inline : true,
+      delay  : false,
+      on     : 'blur',
+
+      fields: {
+        name: {
+          identifier : 'contactName',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.l10n.t('Please enter your name')
+            }
+          ]
+        },
+        company: {
+          identifier : 'company',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.l10n.t('Please enter your company')
+            }
+          ]
+        },
+        country: {
+          identifier : 'country',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.l10n.t('Please enter your country')
+            }
+          ]
+        },
+        address: {
+          identifier : 'address',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.l10n.t('Please enter your billing address')
+            }
+          ]
+        },
+        city: {
+          identifier : 'city',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.l10n.t('Please enter your billing city')
+            }
+          ]
+        },
+        zipCode: {
+          identifier : 'zip',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.l10n.t('Please enter the zip code')
+            }
+          ]
+        },
+        phone: {
+          identifier : 'phone',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.l10n.t('Please enter a phone number.')
+            },
+            {
+              type   : 'regExp',
+              value  : validPhoneNumber,
+              prompt : this.l10n.t('Please enter a valid phone number.')
+            }
+          ]
+        }
+      }
+    };
+  }
+
+  @computed()
+  get countries() {
+    return orderBy(countries, 'name');
+  }
+
+  @action
+  submit() {
```

Comment:
Show a loading UI..
- Either Add a loading icon to the button and disable the button during submit 
- Or show the loading UI over the entire form

See others forms and stick to something that is similar to the others

Reply:
Okay, I used the existing loading UI being used in the other forms.